testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -16,11 +19,19 @@ export default class InstitutionHelperActions {
   }
 
   getLoginUrlForTestEnv() {
-    return `${this.getBaseUrlForTestEnv()}/institution/login`;
+    return `${this.getBaseUrlForTestEnv()}${
+      Cypress.env("END_POINTS").INSTITUTION_LOGIN
+    }`;
   }
 
   getApiUrlForTest() {
-    return `${this.getBaseUrlForTestEnv()}/api/institutions`;
+    return `${this.getBaseUrlForTestEnv()}${
+      Cypress.env("END_POINTS").INSTITUTIONS_API
+    }`;
+  }
+
+  getApiUrlForKeyCloakToken() {
+    return Cypress.env("TEST").KEY_CLOAK_URL;
```

Comment:
I believe that the `KEY_CLOAK_URL` must be defined based on the content of the `ENVIRONMENT`.

Reply:
Yeah. i ll add that approach as part of separate PR